canonical-web-and-design/snapcraft.io
# Add listing page metrics fields

## Done
- Created sections for each group of listing form fields
- Added the "Additional information" form fields

## QA
- Go to https://snapcraft-io-4064.demos.haus/steve-test-snap/listing?show_react_listing=true (or any snap)
- Change the licenses and check that they work as expected

## Issue
Fixes https://github.com/canonical-web-and-design/marketplace-tribe/issues/2651
